CppCoreGuidelines.md

```diff
@@ -19083,6 +19083,7 @@ A `.cpp` file is a form of local scope.
 There is little difference in the opportunities for name clashes in an N-line `.cpp` containing a `using namespace X`,
 an N-line function containing a `using namespace X`,
 and M functions each containing a `using namespace X`with N lines of code in total.
+Also note if you `using namespace X;` with internet based APIs, and that namespace contains a function that is the same name as a internet api function for that platform would result in issues as well. A good example is with `bind` to bind a socket handle to a listening port on the internet apis which in turn would conflict with `X::bind` or if `X` is `std` then `std::bind`. `bind` is a good example here because the internet APIs of all platforms I found contains a `bind` in the global scope (Windows, MAC, Linux, etc).
```

Comment:
That reads like a comment on the rule, rather than an addition to it.
If anything, it could be useful to mention that conflicts such as `std::bind` vs socket `bind` can be resolved by qualifying the latter as `::bind`.